pkg/restmapper/lazyrestmapper.go

```diff
@@ -151,3 +230,13 @@ func getFilteredAPIGroupResources(cl discovery.DiscoveryInterface, groupFilterPr
 	}
 	return result, nil
 }
+
+func (m *LazyRESTMapper) isGroupKnown(groupName string) bool {
+	for _, knownGroup := range m.KnownGroups {
```

Comment:
Just thought, perhaps `KnownGroups` might be a map? Just thought, no strict necessary here.